presto-main/src/test/java/com/facebook/presto/operator/index/TestFieldSetFilteringRecordSet.java

```diff
@@ -34,15 +34,14 @@
 
 public class TestFieldSetFilteringRecordSet
 {
-    private static final TypeManager TYPE_MANAGER = new TypeRegistry();
-    private static final FunctionRegistry FUNCTION_REGISTRY = new FunctionRegistry(TYPE_MANAGER, new BlockEncodingManager(TYPE_MANAGER), new FeaturesConfig());
-
     @Test
     public void test()
     {
+        TypeManager typeManager = new TypeRegistry();
```

Comment:
there is only single test. It is better to reduce scope of such variables. If there any additional tests than it will be extracted.